Src/library/cURL/curl_easy_externals.e

```diff
@@ -17,12 +17,12 @@ feature -- Command
 	init: POINTER
 			-- Declared as curl_easy_init().
 		require
-			dynamic_library_exists: is_dynamic_library_exists
+			dynamic_library_exists: is_dynamic_library_exists or is_static
 		local
 			l_api: POINTER
 		do
 			l_api := api_loader.api_pointer ("curl_easy_init")
-			if l_api /= default_pointer then
+			if l_api /= default_pointer or is_static then
```

Comment:
I don't think we need the `if` statement anymore.

Reply:
I just keep the if statement to check the precondition for the external feature.

Reply:
No need to check the precondition since it is already there. You could write `check l_api /= default_pointer or is_static end` but no need for the if.